packages/markdown/parseMarkdown.js

```diff
@@ -263,7 +263,8 @@ function createRender(context) {
         `<a aria-labelledby="${hash}" class="anchor-link" href="#${hash}" tabindex="-1">`,
         '<svg><use xlink:href="#anchor-link-icon" /></svg>',
         '</a>',
-        `<button title="Post a comment" class="comment-link" data-feedback-hash="${hash}">`,
+        `<span class="display-none" id="${hash}-comment">for ${headingHtml}</span>`,
```

Comment:
The use case of a screenreader on the docs makes no sense in the first place. So I don't think that the overhead of all these extra span DOM nodes is going to fly.

```suggestion
```

Maybe `aria-hidden` would be good enough.